packages/vue-router/src/router.ts

```diff
@@ -40,6 +40,7 @@ export const createIonRouter = (opts: IonicVueRouterOptions, router: Router) =>
      * but we can check to see if the latest routing action
      * was a replace action by looking at the history state.
      */
+    const history = opts.history;
     const replaceAction = history.state.replaced ? 'replace' : undefined;
```

Comment:
```suggestion
     * We need to use opts.history rather than window.history
     * because window.history will be undefined when using SSR.
     */
    const replaceAction = opts.history.state.replaced ? 'replace' : undefined;
```

Can we just add a comment here saying what this is doing? I tried adding it myself, but I am not able to push to your branch.

Reply:
@liamdebeasi thank you I have committed your suggestion.